### PR TITLE
(fix) bugs in merkato, runs with db_init_script

### PR DESCRIPTION
**Pull request by mark-antony1**

### CODE REVIEW

File: db_init_script.py [hunk 1]
```diff
@@ -24,17 +24,19 @@ def main():
 
     base = "BTC"
     coin = "ETH"
-    spread = ".1"
-    merkato = Merkato(configuration, coin, base, spread)
+    spread = .1
```
Comment (nasaWelder) [inline db_init_script.py:27]: i think the proper fix is to make sure that what uses this runs float() on it regardless.
who knows what format the DB returns.

**Review by nasaWelder — CHANGES_REQUESTED** [review]

File: db_init_script.py (hunk 1 above)
Comment (needmoney90) [reply db_init_script.py:27]: Decimal*

Comment (mark-antony1) [reply db_init_script.py:27]: the db has it saved as a float, it will return it as such

Comment (mark-antony1) [reply db_init_script.py:27]: @nasaWelder

Comment (needmoney90) [reply db_init_script.py:27]: We should attempt to use and store Decimal whenever possible

**Pull request merged**